ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -192,6 +197,24 @@ func (p *LocalIgnitionProvider) GetPayload(ctx context.Context, releaseImage str
 	}
 
 	// Extract template files from the MCO image to the MCC input directory
```

Comment:
this comment does not match the code now, does it?

Reply:
No it didn't. I failed to move it when I added my code in. I moved it back down to where it originially was.